src/v2/view-builder/views/AutoRedirectView.js

```diff
@@ -71,12 +71,25 @@ const Body = BaseForm.extend({
     BaseForm.prototype.initialize.apply(this, arguments);
     this.redirectView = this.settings.get('interstitialBeforeLoginRedirect');
     this.model.set('useRedirect', true);
+    if (this.useRedirectButton()) {
+      this.model.set('useRedirectButton', true);
+    }
     this.trigger('save', this.model);
   },
 
+  useRedirectButton() {
+    const idx = this.options.appState.get('idx');
+    const isAndroidOVEnrollment = Util.isAndroidOVEnrollment(this.options.appState.get('authentication'));
+    // Do not show "Open Okta Verify" button for "redirect-idp" remediation
+    //  converted to "success-redirect" with `convertRedirectIdPToSuccessRedirectIffOneIdp()`
+    // (Which can happen if there is a IdP route configured for a user)
+    const isSuccessRedirect = idx?.rawIdxState?.success?.href;
```

Comment:
Without this fix it will show unnecessary page with "Open Okta Verify" button before redirect to IDP (not IDP authenticator).
In my example I've set up Github IdP and routing rule for my user:

https://github.com/user-attachments/assets/ad22f322-fcad-4248-a3cb-00d0eb23b131